Libraries/PeriphDrivers/Include/MAX32680/tpu.h

```diff
@@ -1,993 +0,0 @@
-/**
- * @file
- * @brief   Trust Protection Unit driver.
```

Comment:
Consider putting a deprecated passthrough file here with a warning instead of deleting this file.  This will catch any cases where users have used `tpu.h` and give them a pointer to the new file.  We can delete this in a later release.

Reply:
There's no new file that replaces tpu.h. The TPU (or the CTB) is not supported and there was no source code developed for the ME20.